docs/dev-guide/src/getting-started/building.md

```diff
@@ -16,7 +16,7 @@ For the installation you can refer to [these instructions](https://www.rust-lang
 Once you have rustup, you'll need to do the following.
 1. Install and the nightly rust toolchain specified in [./ci/env](https://github.com/gluwa/creditcoin/blob/dev/ci/env). Creditcoin uses the feature `stdsimd` which was removed in later nightly versions.
 2. Set the newly installed toolchain as the default for your creditcoin checkout
-3. Add the `wasm32-unknown-unknown` target
+3. Add the `wasm32-unknown-unknown` target 
```

Comment:
This isn't going to work. 

The [Auto-update creditcoin-js type definitions](https://github.com/gluwa/creditcoin/pull/1628/commits/80b2bdedd7e07ff75d164a2473f6b344b81cc70c) commit is auto-generated and GitHub does not trigger workflows for commits submitted with github-token to avoid endless loops. 

Revert anything after it. 

The actual satuses can be observed on the previous commit.

Reply:
The reason I was trying to get workflows to trigger is that it seemed the PR couldn't be merged without them.

At least the auto merge seemed to be stalled indefinitely. 

Am I missing something obvious?

Reply:
Alright, I got rid of the commits as requested 👍